README.md

```diff
@@ -214,13 +214,34 @@ A number of options are available to allow you to customise the SDK:
 
   A string of the ID of the container element that the UI will mount to. This needs to be an empty element. The default ID is `onfido-mount`.
 
-- **`language {String} optional`**
-  A string with the language locale tag. At the moment, the SDK can be used in English `en` or Spanish `es`.
-  If `language` is not present it will default to `en`.
+- **`language {String || Object} optional`**
+  The SDK language can be customised by passing a String or an Object. At the moment, we support and maintain translations for English (default) and Spanish, using respectively the following locale tags: `en`, `es`.
+  To leverege one of these two languages, the `language` option should be passed as a string containing a supported language tag.
 
-    ```javascript
-    language: 'es'
-    ```
+  Example:
+  ```javascript
+  language: 'es'
+  ```
+
+  The SDK can be used in a custom language by passing an object containing the locale tag and the custom phrases.
+  The object should include a the following keys:
+    - `locale` (required) : A language tag
```

Comment:
As a follow up to our slack convo - if this prop isn't used yet, it's best to remove it for now (from the docs at least, if not the API) and document it once it does something.